src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/article-templates/landing_page_template.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: 'Landing page template'
+tags:
+translate:
+redirects:
+---
+
+Landing pages are a specialized type of page that serve as the starting pages for various New Relic products. For example, you'll see landing pages for **Application monitoring (APM)** and **Browser monitoring**. 
+
+<Callout variant="tip">
+  
+  This landing page information does not apply to the docs home page.
+  
+</Callout>
+
+
+If you need to create a new landing page, you can either copy an existing landing page, or you can modify the sample landing page shown at the bottom. Let's look at the sections you need for each landing page.
+
+## Front matter
+
+When you insert the front matter, be sure to designate the `type` as `landingPage`. Here's an example:
+
+```
+---
+title: New Relic APM
+type: landingPage
+tags:
+translate:
+redirects:
+---
+```
+
+## Introduction section
+
+After you insert the front matter, insert the introduction, which typically just includes text and an image. If you need help with inserting images, see the example at the bottom.
+
+## Tiles
+
+Tiles are a series of boxes that contain the main subject areas for your product. You should just list these in order you want them to appear, and the cascading style sheet will render them across the page.
+
+Here's an example of a tile: 
+
+```
+<LandingPageTileGrid>
+  <LandingPageTile
+    title="INSERT_YOUR_BOX_TITLE_HERE."
+    icon="fe-INSERT_THE_ICON_NAME"
+  >
+
+Insert tile content here...
+
+</LandingPageTile>
```

Comment:
```suggestion
  >
    Insert tile content here...
  </LandingPageTile>
</LandingPageTileGrid>
```

Since we are using a newer version of MDX, you can now write content like a regular JSX component and don't need the extra space around the content like you do on the dev site. I also noticed the closing `LandingPageTileGrid` tag was missing.

Reply:
Nice catch!